components/listingsPage/filters.js

```diff
@@ -130,6 +130,15 @@ const Filters = () => {
     changeAge({ age: "" });
   };
 
+  console.log(insuranceType);
+  console.log(tripLength);
+  console.log(destination);
+  console.log(cruise);
+  console.log(when);
+  console.log(treatment);
+  console.log(terminal);
+  console.log(equipment);
```

Comment:
These consoles should be deleted (unless they are serving a useful purpose 😄 )

Reply:
> These consoles should be deleted (unless they are serving a useful purpose 😄 )

They are there to show that the values of the form inputs are unchanged when you switch between language

Reply:
OK 👍